ux.symfony.com/templates/cookbook/show.html.twig

```diff
@@ -56,9 +56,37 @@
             </div>
         </div>
 
+        <div class="container container-xl x-4 pt-4 px-md-5 pt-md-5">
+            <div class="cluster">
+                <twig:Badge:Author author="{{ cookbook.author }}" />
+                <twig:Badge:Date label="Published" date="{{ cookbook.publishedAt }}" />
+            </div>
+        </div>
```

Comment:
What do you think about putting the badges at the top of the page between the title and the image ?

Reply:
No opinion